### PR TITLE
Update dependency to iOS 13.1.0 release

### PR DESCRIPTION
**Pull request by erhard-pspdfkit**
Update the project dependency to iOS 13.1.0 and expose new property on existing interfaces.

### CODE REVIEW

File: PSPDFKit.dotnet.iOS.Model/ApiDefinition.cs [hunk 1]
```diff
@@ -6637,13 +6640,19 @@ interface PSPDFXFDFParser {
 
 		[Export ("dataProvider")]
 		IPSPDFDataProviding DataProvider { get; }
+
+		[Export ("ignorePageRotation")]
+		bool ignorePageRotation { get; set; }
```
Comment (dalexsoto) [inline PSPDFKit.dotnet.iOS.Model/ApiDefinition.cs:6645]:
```suggestion
		bool IgnorePageRotation { get; set; }
```

File: PSPDFKit.dotnet.iOS.Model/ApiDefinition.cs [hunk 2]
```diff
@@ -6612,6 +6612,9 @@ interface PSPDFXFDFAnnotationProvider {
 
 		[Export ("loadAllAnnotations")]
 		void LoadAllAnnotations ();
+
+		[Export ("ignorePageRotation")]
+		bool ignorePageRotation { get; set; }
```
Comment (dalexsoto) [inline PSPDFKit.dotnet.iOS.Model/ApiDefinition.cs:6617]:
```suggestion
		bool IgnorePageRotation { get; set; }
```

File: PSPDFKit.dotnet.iOS.Model/ApiDefinition.cs [hunk 3]
```diff
@@ -6637,13 +6640,19 @@ interface PSPDFXFDFParser {
...
 	}
 
 	[BaseType (typeof (NSObject))]
 	interface PSPDFXFDFWriter {
 
 		[Export ("writeAnnotations:toDataSink:documentProvider:error:")]
 		bool WriteAnnotations (PSPDFAnnotation [] annotations, IPSPDFDataSink dataSink, PSPDFDocumentProvider documentProvider, [NullAllowed] out NSError error);
+
+		[Export ("ignorePageRotation")]
+		bool ignorePageRotation { get; set; }
```
Comment (dalexsoto) [inline PSPDFKit.dotnet.iOS.Model/ApiDefinition.cs:6655]:
```suggestion
		bool IgnorePageRotation { get; set; }
```

**Review by dalexsoto — CHANGES_REQUESTED** [review]
Looking good!

**Review by dalexsoto — APPROVED** [review]

**Review by matej — APPROVED** [review]

**Pull request merged**